machine/netberg/netberg_common_rangeley/installer/install-platform

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+# Overrides install.sh
+check_machine_image()
+{
+    [ -x "/usr/bin/mb_detect" ] && {
+        platform=$(/usr/bin/mb_detect -p)
+    }
+    [ -x "/usr/bin/mbdetect" ] && {
+        platform=$(/usr/bin/mbdetect -p)
+    }
```

Comment:
Again, why support both names?  Since this is the first I would only support one name.